src/plugins/linked.js

```diff
@@ -162,13 +162,23 @@ export default class LinkedPlugins extends Manager {
 
   async _install (p: string) {
     if (!this._needsInstall(p)) return
+    await this._forceInstall(p)
+  }
+
+  async _forceInstall (p: string) {
```

Comment:
could we instead make this `async _install (p: string, force: boolean = false)`?